src/router-view.js

```diff
@@ -1,13 +1,34 @@
 import {Container, inject} from 'aurelia-dependency-injection';
 import {ViewSlot, ViewLocator, customElement, noView, BehaviorInstruction} from 'aurelia-templating';
+import {bindable} from 'aurelia-framework';
```

Comment:
This library doesn't depend on aurelia-framework. We'll need to import `bindable` from aurelia-templating.